### PR TITLE
RDS Postgres Module

### PR DESCRIPTION
**Pull request by grebois**

### CODE REVIEW

File: terraform-modules/aws/postgres/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+| <a name="input_password"></a> [password](#input\_password) | Password for the master DB user. Note that this may show up in logs, and it will be stored in the state file | `string` | `null` | no |
+| <a name="input_storage_encrypted"></a> [storage\_encrypted](#input\_storage\_encrypted) | Specifies whether the DB instance is encrypted | `bool` | `true` | no |
+| <a name="input_storage_type"></a> [storage\_type](#input\_storage\_type) | One of 'standard' (magnetic), 'gp2' (general purpose SSD), or 'io1' (provisioned IOPS SSD). The default is 'io1' if iops is specified, 'gp2' if not. | `string` | `"gp2"` | no |
+| <a name="input_tags"></a> [tags](#input\_tags) | n/a | `map(any)` | <pre>{<br>  "ops_env": "staging",<br>  "ops_managed_by": "terraform",<br>  "ops_owners": "devops",<br>  "ops_source_repo": "kubernetes-ops"<br>}</pre> | no |
+| <a name="input_username"></a> [username](#input\_username) | Username for the master DB user | `string` | `null` | no |
+| <a name="input_vpc"></a> [vpc](#input\_vpc) | VPC where the rds and security group will be created | `any` | n/a | yes |
+
+## Outputs
+
```
Comment (sekka1) [inline terraform-modules/aws/postgres/README.md:41]: I think this is inaccurate: https://github.com/ManagedKube/kubernetes-ops/pull/148/files#diff-187ff3e031edb6a9c75ac58cc22fe7b107796b6a3d59deecfb055640294e98c1R1

Comment (grebois) [reply terraform-modules/aws/postgres/README.md:41]: indeed, it is outdated

File: terraform-modules/aws/postgres/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+No outputs.
+
+<small><i><a href='https://github.com/terraform-docs/terraform-docs'>Generated with terraform-docs</a></i></small>
```
Comment (sekka1) [inline terraform-modules/aws/postgres/README.md:44]: can you put in a head section "##" with this...it looks like it is with the outputs head section

File: terraform-modules/aws/postgres/README.md (hunk 1 above)
Comment (grebois) [reply terraform-modules/aws/postgres/README.md:41]: Updated.

File: terraform-modules/aws/postgres/main.tf [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  username = var.username
+  password = var.password
+  port     = 5432
+
+  multi_az               = true
+  subnet_ids             = var.vpc.outputs.private_subnets
+  vpc_security_group_ids = [module.security_group.security_group_id]
+
+  maintenance_window              = "Mon:00:00-Mon:03:00"
+  backup_window                   = "03:00-06:00"
```
Comment (sekka1) [inline terraform-modules/aws/postgres/main.tf:54]: these items should be a variable

File: terraform-modules/aws/postgres/main.tf [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    {
+      name  = "autovacuum"
+      value = 1
+    },
+    {
+      name  = "client_encoding"
+      value = "utf8"
+    }
+  ]
+
```
Comment (sekka1) [inline terraform-modules/aws/postgres/main.tf:76]: all of the above should be a parameter....especially the `paramters' list...everyone will have their own settings they want.  Make the defaults what you have above though so the user dont have to set it unless they want to.

File: terraform-modules/aws/postgres/main.tf [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+  db_option_group_tags = {
+    "Sensitive" = "low"
+  }
+  db_parameter_group_tags = {
+    "Sensitive" = "low"
+  }
+  db_subnet_group_tags = {
+    "Sensitive" = "high"
+  }
+}
```
Comment (sekka1) [inline terraform-modules/aws/postgres/main.tf:87]: what are these group tags for?

File: terraform-modules/aws/postgres/variables.tf [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+  type        = string
+  default     = null
+}
+variable "tags" {
+  type = map(any)
+  default = {
+    ops_env         = "staging"
+    ops_managed_by  = "terraform",
+    ops_source_repo = "kubernetes-ops",
+    ops_owners      = "devops",
```
Comment (sekka1) [inline terraform-modules/aws/postgres/variables.tf:75]: there should be a tag named: `ops_source_repo_path`.  

Again from a user's experience point of view.  Im in the console looking at this RDS and want to make a change.  Ok cool..it is managed by terraform.  It points me to the repo but this repo has a lot of stuff.  Now i have to go and figure out which actual thing instantiated it.  With the path, i know exactly what created this TF and its resources. 

I personally have ran into this so many times looking at other peoples TF infra...i have no idea where this stuff was created from and you have to go ask the sysadmin and this person will tell you.....not a good workflow.

File: terraform-modules/aws/vpc/README.md [hunk 7]
```diff
@@ -1,3 +1,49 @@
...
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| <a name="output_private_subnets"></a> [private\_subnets](#output\_private\_subnets) | A list of private subnets |
+| <a name="output_public_subnets"></a> [public\_subnets](#output\_public\_subnets) | A list of public subnets |
+| <a name="output_vpc_cidr_block"></a> [vpc\_cidr\_block](#output\_vpc\_cidr\_block) | The CIDR block of the VPC |
+| <a name="output_vpc_id"></a> [vpc\_id](#output\_vpc\_id) | The ID of the VPC |
+
+<small><i><a href='https://github.com/terraform-docs/terraform-docs'>Generated with terraform-docs</a></i></small>
```
Comment (sekka1) [inline terraform-modules/aws/vpc/README.md:49]: same comment here...put a heading section on this as it looks like it is in the output section.

**Review by sekka1 — APPROVED** [review]
lgtm

**Pull request merged**